### PR TITLE
chore: remove unused env and new create unique task

### PR DESCRIPTION
**Pull request by dni**
remove old scheduled tasks approach

breaking change should be min version `0.12.3`

### CODE REVIEW

**Comment by motorina0** [comment]
> breaking change should be min version `0.12.3`

please update `"min_lnbits_version"` in `config.json`

**Comment by dni** [comment]
done

**Comment by dni** [comment]
no, it does not you need to manually fix the PR in that repo right now, duplicate it and add another min version it.

File: views_api.py [hunk 1]
```diff
@@ -182,17 +181,6 @@ async def api_check_fiat_rate(currency):
     return {"rate": rate}
 
 
-@lnurlp_ext.delete("/api/v1", dependencies=[Depends(check_admin)])
-async def api_stop():
```
Comment (motorina0) [inline views_api.py:186]: - the endpoint is being removed without a `lnurlp_stop` being added

Comment (dni) [reply views_api.py:186]: damn, yes you are so right, still important for dynamic stopping, i was just thinking about restarting

Comment (dni) [reply views_api.py:186]: done

File: __init__.py [hunk 2]
```diff
@@ -36,5 +35,14 @@ def lnurlp_renderer():
...
+
+
+def lnurlp_stop():
+    for task in scheduled_tasks:
+        task.cancel()
+
 def lnurlp_start():
-    create_permanent_unique_task("lnurlp", wait_for_paid_invoices)
+    task = create_permanent_unique_task("lnurlp", wait_for_paid_invoices)
+    task.append(task)
```
Comment (motorina0) [inline __init__.py:48]: image

Comment (motorina0) [reply __init__.py:48]:
```suggestion
    scheduled_tasks.append(task)
```

**Review by motorina0 — APPROVED** [review]

**Pull request merged**